File: CPKINGDOM/CPKINGDOM.Portal/ClientApp/src/app/models/staff.ts
export class Staff {
    id: number;
    firstName: string;
    middleName: string;
    lastName: string;
    address: string;
    contactNo: string;
    roleId: number;
    roleName: string;
    fullName: string;
    constructor() {
        this.id = 0;
        this.firstName = '';
        this.middleName = '';
        this.lastName = '';
        this.address = '';
        this.contactNo = '';
        this.roleId = 0;
        this.roleName = '';
        this.fullName = this.firstName + ' ' + this.lastName;
    }
}

File: CPKINGDOM/CPKINGDOM.Api/Models/Staff.cs
namespace CPKINGDOM.Api.Models
{
    public class Staff
    {
        public int Id { get; set; }
        public string FirstName { get; set; }
        public string MiddleName { get; set; }
        public string LastName { get; set; }
        public string Address { get; set; }
        public string ContactNo { get; set; }
        public int RoleId { get; set; }
        public string RoleName { get; set; }
        public string FullName { get; set; }
    }
}